src/applications/personalization/rated-disabilities/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "appName": "Rated Disabilities",
+  "entryFile": "./rated-disabilities-entry.jsx",
+  "entryName": "disability/my-rated-disabilities",
```

Comment:
@micahchiang The `entryName` will be used as the JS bundle's file name, so it shouldn't have slashes like this. That's why the build is failing.

```
/application/node_modules/enhanced-resolve/node_modules/memory-fs/lib/MemoryFileSystem.js:80
			throw new Error("Cannot readFile on directory '" + _path + "'");
			^

Error: Cannot readFile on directory '/application/build/vagovdev/generated/disability'
```

Reply:
@ncksllvn - updated entryname to remove slashes.

Reply:
I think the build passed now 🎉